core/src/main/java/com/netease/arctic/utils/map/RocksDBBackend.java

```diff
@@ -50,17 +50,29 @@
 
 public class RocksDBBackend {
   private static final Logger LOG = LoggerFactory.getLogger(RocksDBBackend.class);
-  private static final String BACKEND_BASE_DIR = System.getProperty("rocksdb.dir");
+  private static final String BACKEND_BASE_DIR = System.getProperty("java.io.tmpdir");
   private static final ThreadLocal<RocksDBBackend> instance =
           new ThreadLocal<>();
 
   public static RocksDBBackend getOrCreateInstance() {
     RocksDBBackend backend = instance.get();
     if (backend == null) {
-      backend = create();
+      backend = create(BACKEND_BASE_DIR);
     }
     if (backend.closed) {
-      backend = create();
+      backend = create(BACKEND_BASE_DIR);
+      instance.set(backend);
+    }
+    return backend;
+  }
+
+  public static RocksDBBackend getOrCreateInstance(String backendBaseDir) {
+    RocksDBBackend backend = instance.get();
+    if (backend == null) {
+      backend = create(backendBaseDir);
+    }
+    if (backend.closed) {
+      backend = create(backendBaseDir);
```

Comment:
The instant is share with thread, so the backendBaseDir can not be modify once created in the same thread

Reply:
> The instant is share with thread, so the backendBaseDir can not be modify once created in the same thread

I think the backendBaseDir should be modified with new backendBaseDir